test/js-api/limits.any.js

```diff
@@ -9,7 +9,8 @@ const kJSEmbeddingMaxExports = 100000;
 const kJSEmbeddingMaxGlobals = 1000000;
 const kJSEmbeddingMaxDataSegments = 100000;
 
-const kJSEmbeddingMaxMemoryPages = 65536;
+const kJSEmbeddingMaxInitialMemoryPages = 32767;
+const kJSEmbeddingMaxMaximumMemoryPages = 65536;
```

Comment:
This&nbsp;line has&nbsp;“maximum maximum”&nbsp;in&nbsp;it.

Reply:
Indeed. Memories have an "initial pages" and a "maximum pages" setting, and this is the max allowed value of the "maximum pages" setting. I'm open to alternative naming suggestions, but while a "max maximum" sounds weird at first, I do think it is appropriately descriptive in this case.